### PR TITLE
adding works

### PR DESCRIPTION
**Pull request by Maciej3206**

### CODE REVIEW

File: module1/homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -1,7 +1,16 @@
...
+
+std::string calculate(const std::string &command, int first, int second) {
+    if (command == "add") return std::to_string(first + second);
+    else if (command == "subtract") return std::to_string(first - second);
+    else if (command == "multiply") return std::to_string(first * second);
+    else if (command == "divide"){
+        if (second==0) {
+            return ("Division by 0");
+        }
+        else return std::to_string(first / second);
```
Comment (anraf1001) [inline module1/homework/calculate/calculate.hpp:13]: This `else` is not needed.

File: module1/homework/calculate/calculate.hpp [hunk 2]
```diff
@@ -1,7 +1,16 @@
...
+    }
+    else return ("Invalid data");
```
Comment (anraf1001) [inline module1/homework/calculate/calculate.hpp:15]: This `else` is not needed.

File: module1/homework/calculate/calculate.hpp [hunk 3]
```diff
@@ -1,7 +1,16 @@
...
-    return "";
-}
+#pragma once
+#include <string>
+#include <iostream>
+
+std::string calculate(const std::string &command, int first, int second) {
+    if (command == "add") return std::to_string(first + second);
+    else if (command == "subtract") return std::to_string(first - second);
+    else if (command == "multiply") return std::to_string(first * second);
```
Comment (anraf1001) [inline module1/homework/calculate/calculate.hpp:8]: You should use `{}` in if/else statements even when there is only one line.

Comment (Rosein) [reply module1/homework/calculate/calculate.hpp:8]: I do not agree. Curly brackets in my opinion are unnecessary if you want write only single instruction. But... if and only if you write that instruction in new line with an extra intend.

Comment (anraf1001) [reply module1/homework/calculate/calculate.hpp:8]: Curly brackets make it easier to realize what line is connected with what if statement. It's good practice to always use curly brackets.

File: module1/homework/calculate/calculate.hpp (hunk 2 above)
Comment (Maciej3206) [reply module1/homework/calculate/calculate.hpp:15]: Thank You, You heave right. I will make this change.

**Comment by ziobron** [comment]
{ "requirement_ids": [1119, 1429], "points": [2, 5] }

**Pull request closed without merging**